sentry-uitest/sentry-uitest-android-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import net.ltgt.gradle.errorprone.errorprone
+
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+android {
+    compileSdk = Config.Android.compileSdkVersion
+
+    defaultConfig {
+        applicationId = "io.sentry.uitest.android.benchmark"
+        minSdk = Config.Android.minSdkVersionNdk
+        targetSdk = Config.Android.targetSdkVersion
+        versionCode = 1
+        versionName = "1.0.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        // Runs each test in its own instance of Instrumentation. This way they are isolated from
+        // one another and get their own Application instance.
+        // https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner#enable-gradle
+        // This doesn't work on some devices with Android 11+. Clearing package data resets permissions.
+        // Check the readme for more info.
+//        testInstrumentationRunnerArguments["clearPackageData"] = "true"
+    }
+
+    testOptions {
+        execution = "ANDROIDX_TEST_ORCHESTRATOR"
+    }
+
+    signingConfigs {
+        getByName("debug") {
+            storeFile = rootProject.file("debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+        }
+    }
+
+    testBuildType = System.getProperty("testBuildType", "debug")
+
+    buildTypes {
+        getByName("debug") {
+            isDebuggable = false
+            isMinifyEnabled = true
+            isMinifyEnabled = true
+            signingConfig = signingConfigs.getByName("debug")
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "benchmark-proguard-rules.pro")
+        }
+        getByName("release") {
+            isDebuggable = false
+            isMinifyEnabled = true
+            isShrinkResources = true
+            signingConfig = signingConfigs.getByName("debug") // to be able to run release mode
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "benchmark-proguard-rules.pro")
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+
+    lint {
+        warningsAsErrors = true
+        checkDependencies = true
+
+        // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks.
+        checkReleaseBuilds = false
+    }
+
+    variantFilter {
+        if (Config.Android.shouldSkipDebugVariant(buildType.name)) {
+            ignore = true
+        }
```

Comment:
Do we need the `test` variant for your tests on CI?

Reply:
We will need `release` variant only. 
Also, if `debug` variant is built on ci, everything breaks, as it expects its dependency modules (like sentry-android) to be `debug`, too, but they also skip the debug variant build on ci.

Reply:
Then I don't see the need of configuring `getByName("debug")` in the first place.

Reply:
just because doing so, you don't have to manually select the `release` variant of the other modules if you want to run the benchmarks locally (android studio complains about `variants conflict`), and then reselect the `debug` variant if you want to work on other modules.
Or am I missing something?

Reply:
If AS complaints about `variants conflict`, all good, I never experienced this error so I was not sure if it was needed.